src/core/upload/index.ts

```diff
@@ -96,15 +96,15 @@ export async function checkUploadReadiness(options: UploadReadinessOptions): Pro
   onProgress?.({ type: 'checking-balances' })
 
   const filStatus = await checkFILBalance(synapse)
-  const usdfcBalance = await checkUSDFCBalance(synapse)
+  const walletUsdfcBalance = await checkUSDFCBalance(synapse)
```

Comment:
i'm going to move these rename changes to a separate PR so it's not muddying up these changes.